Source/WebCore/accessibility/AccessibilityNodeObject.cpp

```diff
@@ -377,6 +377,8 @@ AccessibilityRole AccessibilityNodeObject::determineAccessibilityRoleFromNode(Tr
         return AccessibilityRole::LandmarkNavigation;
     if (node()->hasTagName(asideTag))
         return AccessibilityRole::LandmarkComplementary;
+    if (node()->hasTagName(searchTag))
+        return AccessibilityRole::LandmarkSearch;
```

Comment:
@twilco I could not discern an an ordering in this file. Happy to move this around though.